### DISCUSSION TITLE
The documentation link isn't working on the main page

### DISCUSSION THREAD

**Discussion by plppp2001** [CommunityToolkit/Datasync#62] [General]
The documentation link isn't working on the main page

**Comment by MichiganMan83** [comment]
This is mega-confusing. The MAUI docs for Azure Mobile Apps says "This product is retired. For a replacement for projects using .NET 8 or later, see the [Community Toolkit Datasync library](https://aka.ms/azure-mobile-apps/docs)." That points to this. But the documentation link goes into space. So what's the scoop? Do we have one solution that is no longer supported and one solution that is not yet ready? If we're developing an app that requires offline sync right now, what are we supposed to use? There doesn't appear to be any solid direction on what us poor devs are supposed to do at that point. It all looks awesome, by the way! We just need to know what path to follow. Hoping you can shed some light on the subject for us. Thanks!

**Reply by adrianhall** [reply]
I seem to have answered this multiple times now.

Azure Mobile Apps is retired.  I could not make it conform to the security requirements that Microsoft puts on software that is going to be tagged with their name as a side project.  It's still completely functional - it's not going to get any updates from now on.  If you are using .NET 6, this is actually the library to target.

Community Datasync Toolkit is the way forward.  I am working feverishly to release it, but it isn't ready yet.  I'm about a week away from being code complete.  If you are using .NET 8, this is the library you want to follow.

Once I am code complete, I'm going to leave the code alone so everyone can bash on it while I write the documentation.  Once everything is done, I'll release the NuGet packages.

**Reply by MichiganMan83** [reply]
Adrian... First, apologies that I didn't find those previous answers. The official Microsoft docs just kind of put us in an endless loop without seeing your explanations, and it's not clear exactly how this stuff all fits into the bigger ecosystem. So I appreciate you clarifying the situation here. Second, this timeline sounds great. It's not like I'm getting my project done in the next few weeks, so your timeline is just fine for me, and hopefully many others. Third, this work is fantastic! It's something that is incredibly important for my app, and I can only assume many, many others. I hope that I can be in in a place to help test the code and provide any feedback that might help out. Again, much appreciation for all you're doing. Cheers...

**Comment by adrianhall** [comment]
I've updated the documentation now with a first-pass of the documentation requirements.

I'm currently working on releasing the first "beta" version of the new library.  So - coming this week.
